portfolio/src/main/java/com/google/sps/data/Account.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+public class Account{
+
+  private boolean isLoggedIn;
+  private boolean isAdmin; 
+  private String userEmail;
+  private String loginLogoutURL; // Link to either login or logout page, depending on isLoggedIn.
+  private String username;
```

Comment:
Just a note about these (no changes necessary):

If you had some attributes of an Account that you wanted to make immutable, you could consider declaring them "final." I don't think that's necessary here since you might want to allow the user to change their email/username/etc, but I wanted to call it out.

https://www.educative.io/edpresso/what-is-the-final-keyword-in-java